rest/src/main/kotlin/rys/rest/service/impl/UserServiceImplementation.kt

```diff
@@ -1,12 +1,12 @@
-package rys.ajaxpetproject.service.impl
+package rys.rest.service.impl
```

Comment:
Mostly, a project has a basic package like: `systems.ajax`, `com.google`, `com.github`

Ok, you have chosen `rys` as root. Then choose the next basic subdomain, for example `ajaxpetproject` whatever. Just choose the application name.

so, your basic, mandatory and common package for all project would be something like `rys.ajaxpetproject`.

Then for the `rest` basic package would be `rys.ajaxpetproject.rest`, for nats `rys.ajaxpetproject.nats`, etc.

Don't make your task harder